src/main/java/com/newrelic/telemetry/opentelemetry/export/SpanBatchAdapter.java

```diff
@@ -73,44 +71,13 @@ private static Attributes generateSpanAttributes(SpanData span) {
 
   private static Attributes addPossibleInstrumentationAttributes(
       SpanData span, Attributes attributes) {
-    InstrumentationLibraryInfo instrumentationLibraryInfo = span.getInstrumentationLibraryInfo();
-    if (instrumentationLibraryInfo != null) {
-      if (instrumentationLibraryInfo.getName() != null
-          && !instrumentationLibraryInfo.getName().isEmpty()) {
-        attributes.put("instrumentation.name", instrumentationLibraryInfo.getName());
-      }
-      if (instrumentationLibraryInfo.getVersion() != null
-          && !instrumentationLibraryInfo.getVersion().isEmpty()) {
-        attributes.put("instrumentation.version", instrumentationLibraryInfo.getVersion());
-      }
-    }
-    return attributes;
+    attributes = CommonUtils.populateLibraryInfo(attributes, span.getInstrumentationLibraryInfo());
```

Comment:
nitpick: prefer not to reassign input parameter.

Reply:
agreed. fixing

Reply:
fixed.